kubernetes-1.31.yaml

```
@@ -195,7 +195,7 @@ subpackages:
     pipeline:
       - runs: |
           mkdir -p ${{targets.subpkgdir}}/usr/local/bin
-          ln -s kube-proxy-${{vars.kubernetes-version}} ${{targets.subpkgdir}}/usr/local/bin/kube-proxy
+          ln -sf /usr/bin/kube-proxy ${{targets.subpkgdir}}/usr/local/bin/kube-proxy
```

Nit: the name of the binary also contains major-minor version suffix.
```suggestion
          ln -sf /usr/bin/kube-proxy-${{vars.kubernetes-version}} ${{targets.subpkgdir}}/usr/local/bin/kube-proxy
```